userspace/libsinsp/cri.h

```diff
@@ -41,13 +41,97 @@ limitations under the License.
 namespace libsinsp {
 namespace cri {
 
-// these shouldn't be globals but we still need references to *the* CRI runtime
-extern std::vector<std::string> s_cri_unix_socket_paths;
-extern int64_t s_cri_timeout;
-// TODO: drop these 2 below
```

Comment:
Drop these 2 below? Anyone knows what this todo item actually means? Can we address it?

Reply:
Taking a look around,  they could have a legitimate use given that they gives useful information about the actual socket path used by the container runtime and its type. In my opinion, we should keep them